test/WorkItemBuiltins/get_global_offset-push-constant.cl

```diff
@@ -0,0 +1,27 @@
+// RUN: clspv -global-offset -descriptormap=%t.dmap %s -o %t.spv
```

Comment:
Same here, a version for an out of range access.

Reply:
That is covered by the `get_global_id(3)` test above IMHO since `get_global_id` calls `get_global_offset`. The only thing that makes it into the SPIR-V is a store of a 0 constant.